recipes/liblbfgs/build.sh

```diff
@@ -0,0 +1,6 @@
+./configure --prefix $PREFIX 
+make
+make install
```

Comment:
is there a `make test` as well or something similar?

Reply:
there is a sample binary. I tried to copy it to `$PREFIX/bin` and execute it in the test .. which failed. 

I could still execute it in the test..

Reply:
Reason for the tailure during the test is some weird linking problem that I don't understand

Reply:
So I added the additional test call to build.sh. Seems a good compromise.